containers/pre_built_workers/python/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM l.gcr.io/google/bazel:latest
+
+RUN mkdir -p /pre
+WORKDIR /pre
+
+ARG REPOSITORY=grpc/grpc
+ARG GITREF=master
+
+RUN git clone https://github.com/$REPOSITORY.git .
+RUN git submodule update --init
+RUN git checkout $GITREF
+
+# Save commit sha for debug use
+RUN echo 'COMMIT SHA' > GRPC_GIT_COMMIT.txt
+RUN git rev-parse $GITREF >> GRPC_GIT_COMMIT.txt
+
+RUN mkdir /executables
+RUN bazel --output_user_root=/executables build --nobuild_runfile_links --nobuild_runfile_manifests --build_python_zip //src/python/grpcio_tests/tests/qps:qps_worker
```

Comment:
As we have spoke in the meeting, you might want to have -c opt option. See reference: https://github.com/grpc/grpc/blob/4768210ec860fd9a5ecba14cf11de147c32873fd/tools/run_tests/performance/build_performance.sh#L72-L77

Reply:
Other than that LGTM!

Reply:
Done! Thanks for reviewing:DD